extensions/oidc/runtime/src/main/java/io/quarkus/oidc/RefreshToken.java

```diff
@@ -0,0 +1,14 @@
+package io.quarkus.oidc;
```

Comment:
Previously the package was `io.quarkus.oidc.runtime` 

google groups thread mentions change in package - *though in a new package*
What's the reason for package change ?

Reply:
@rsvoboda sure, it is mean tot be part of a public API, so it needs to be located outside of the `runtime`. 
As I mentioned in the google thread, in fact `RefreshToken` was duplicated both in `io.quarkus.oidc` and `io.quarkus.runtime` before I removed both of them with an earlier PR, so this PR actually restores the one in `io.quarkus.oidc` to be in the same package as `AccessTokenCredential`/etc.
Does it answer your question ? I know, it looks not good when we see a package name change. But it is the right fix and is just in time.

Reply:
thanks for details